eclair/docs/index.md

```diff
@@ -0,0 +1,108 @@
+# ⚡️ Eclair ⚡️
+
+## Helping AI Agents access the world's data
+
+Eclair is a set of agentic tools that helps AI assistants (like Gemini and Claude) discover, download, and use datasets to answer questions based on factual data. Such AI models are trained on a lot of data from the internet and other sources, but usually not on the millions of large detailed structured datasets that humanity has gathered. That also means that they can't correctly answer questions about that data, which limits their scientific and analytical potential.
+
+With Eclair, AI models can answer those questions by easily finding and accessing millions of datasets available all over the world, analyze this data on the fly to give accurate answers, reason about it and combine it with other data, as well as generate detailed analyses (e.g. notebooks) and custom UIs to explore the data in depth. It's like having your own personal, lightning-fast data scientist.
+
+<img src="images/eclair-diagram2.png" alt="Eclair Diagram" style="max-width: 100%; height: auto;"/>
+
+Eclair does this by combining three recent technologies:
+
+* **[Model Context Protocol (MCP)](https://modelcontextprotocol.io/docs/getting-started/intro)** - Gives AI models easy access to millions of datasets available online (or your own), and tools to work effectively with that data.
+
+* **[Croissant](https://github.com/mlcommons/croissant)** - A metadata standard to help AI models understand what information each dataset contains and how to download and extract that information.
+
+* **[AI assistants](https://modelcontextprotocol.io/clients)** - That can 'think' about how to answer your questions, find relevant datasets, write code to download and analyze this data, and generate anything from direct textual answers to detailed notebooks, voice summaries, and custom user interfaces.
+
+Most importantly, Eclair aims to create an **open, global ecosystem** of data sources, agentic tools, and other MCP servers that can be accessed by any AI agent and extended by anyone. You can also run Eclair servers locally, connect it to your own data sources, extend them with more advanced data analysis tools, actively generate data on the fly, or practically anything that you can imagine.
+
+Get involved! With Eclair, your AI assistants can discover, analyze, and visualize real data to give you accurate, factual answers. Your contributions help make dataset discovery better for everyone. If you have questions about contributing or improving Eclair, please don't hesitate to ask in GitHub Discussions or open an issue.
+
+!!! info "Fun Fact"
+    Like 🥐 Croissant, Éclair was born in Paris. It's both a [delicious pastry](https://en.wikipedia.org/wiki/%C3%89clair) and the French word for ⚡️ lightning ⚡️.
```

Comment:
The Austrian's might quibble about the Croissant! https://www.google.com/search?q=croissant+history&oq=croissant+history&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQABiABDIHCAIQABiABDIHCAMQABiABDIHCAQQABiABDIICAUQABgWGB4yCAgGEAAYFhgeMggIBxAAGBYYHjIICAgQABgWGB4yBggJEC4YQNIBCDM3MjFqMGoxqAIAsAIA&sourceid=chrome&ie=UTF-8

Reply:
But of course I mean MLC croissant. I'm sure that the Austrians can't quibble about that :)